deploy/docker/kiali-server.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+if [[ ! "${SERVER_WEB_ROOT}" =~ ^\/[a-zA-Z\d_\$]*$ ]]; then
+   >&2 echo "SERVER_WEB_ROOT invalid format. Example, /kiali"
+   exit 1
+fi
+
+# generate new env file from env vars
+./env2js.sh | tee ./console/env.js
```

Comment:
see my comments above about this - there is no need to make it more complicated than it has to be - get rid of a second script that has to be managed - just do it all here. Unless there is something I am missing about why env2js.sh is needed to be a separate script file.

Reply:
e.g. get rid of env2js.sh and replace line 9 with these 3 lines:

```
if [ ! -z ${SERVER_WEB_ROOT} ]; then
  echo window.WEB_ROOT=${SERVER_WEB_ROOT} > ./console/env.js
fi
```

Reply:
I made `env2js.sh` self-contained to make easier/cleaner to support additional vars in the future. For example, if we need to separate frontend and backend, the backend api url can be configured the same way.  Your thoughts?

Reply:
and the first line in the container log `window.WEB_ROOT='/';` is produced by [kiali-server.sh](https://github.com/kiali/kiali/blob/90522001a22a2fa490f671927b27d6fda77db244/deploy/docker/kiali-server.sh#L9) script prior to the go app.

Reply:
I still would get rid of it. Until we actually have a need/requirement for additional env vars, this just makes this more confusing and complicated.

Reply:
removed env2js.sh and -pushed the PR